src/panel/login_masq.js

```diff
@@ -0,0 +1,66 @@
+import LoginMasqPanelView from '../views/login_masq.dot'
+import Panel from "../libs/panel";
+import Store from "../adapters/store"
+import Error from '../adapters/error'
+import nconf from "../../local_modules/nconf_getter";
+let moduleConfig = nconf.get().store
+
+
+export default class LoginMasqPanel {
+  constructor() {
+    this.panel = new Panel(this, LoginMasqPanelView)
+    if(moduleConfig.name === 'masq') {
+      this.store = new Store()
+      this.isActive = false
+      this.loggingIn = false
+      this.username = null
+      this.profileImage = null
+      listen('login_panel__show', () => {
+        this.panel.animate(.25, '.login_masq_panel', {top : '100px'})
+      })
+
+      this.isLoggedIn = false
+      this.store.isLoggedIn().then(async (b) => {
+        this.isLoggedIn = b
+        if (this.isLoggedIn) {
+          const { username, profileImage } = await this.store.getUserInfo()
+          this.username = username
+          this.profileImage = profileImage
+        }
+        this.panel.update()
+      })
+    }
+  }
+
+  async login() {
+    this.loggingIn = true
+    this.panel.update()
+    try {
+      await this.store.login()
+      this.isLoggedIn = await this.store.isLoggedIn()
+      if (this.isLoggedIn) {
+        const { username, profileImage } = await this.store.getUserInfo()
+        this.username = username
+        this.profileImage = profileImage
+      }
+    } catch(e) {
+      Error.sendOnce('login_masq', 'login', 'error loggingIn masq', e)
```

Comment:
i think this error is already handle in store 
i prefer avoid sending 2 messages with the same error

Reply:
okay ! removed the second error logging in panel/login_masq as the first one can be more specific with the name of the store which is used